about.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet'>
+    <link rel="stylesheet" href="about.css" type="text/css" media="all" />
+    <title>About Page</title>
+</head>
+<body class="body">
+  <header id="header">
+        <div class="so-container">
+        <div class="socials">
+            <img id="facebook" src="./images/socials/facebook.svg" alt="Facebook Handle.">
+            <img id="twitter" src="./images/socials/twitter.svg" alt="Twitter Handle.">
+            <ul class="nav-list">
+                <li id="top-1">English</li>
+                <li id="top-2">My Page</li>
+                <li id="top-3">Logout</li>
+            </ul>
+            </div>
+            </div>
+  <nav class="a-nav">
+     <img id="ham-menu" src="./images/ham.png" alt="Hamburger Menu" />
+     <img id="a-logo" src="./images/nba-logos/nba-logo.svg" alt="Logo">
+     <div id="this-menu">
+     <img id="a-exit" src="./images/close-button.png" alt="Exit Menu" />
+     <ul id="nav-menu">
+        <li id="home"><a class="link" href="index.html">Home</a></li>
+        <li id="menu-1"><a class="link" href="about.html">About</a></li>
+        <li id="menu-2">Program</li>
+        <li id="menu-3">Join</li>
+        <li id="menu-4">Sponsor</li>
+        <li id="menu-5">News</li>
+        <li id="menu-6">Others</li>
```

Comment:
- [ ] The navigation bar is supposed to look the same on both the home page and the about page. Yours looks different and the aligned is also different please try to correct on that too 
**About**
![image](https://user-images.githubusercontent.com/90222110/183120654-d61f3a7d-14f9-439d-a9f4-7f51e9fe2cce.png)

**Home**
![image](https://user-images.githubusercontent.com/90222110/183120973-c9941645-d603-491a-ba9c-0cb7214ba47e.png)

Reply:
- [ ] Please make sure to implement these changes, as requested by the previous reviewer.